### PR TITLE
created garden plot thumbnails for garden index plot layout for garden

### LINKED ISSUE #17: User views garden plans on homepage
## User views garden plans on homepage

As a user
In order to see an overview of my designs for my gardens
I want to view my garden plans on the homepage

Usage:
- The home screen shows the list of plans, with a summary image of the plan and a link to the plan.
- The summary image should be generated via. a helper (e.g. GardenHelper) method that we will later fill in with a more detailed rendering of the garden.  For now, a table with the appropriate number of rows and columns will suffice.

### PR DESCRIPTION
**Pull request by ccampb13**
@elizabrock completed instructions to add garden plot to garden/index and edit

resolves #17

### CODE REVIEW

File: app/views/gardens/index.html.haml [hunk 1]
```diff
@@ -6,6 +6,11 @@
     %ul
       - current_user.gardens.each do |garden|
         %li= link_to garden.name, edit_garden_path(garden)
+        %table{:border => 1}
+          - "#{garden.width}".to_i.times do
```
Comment (elizamarcum) [inline app/views/gardens/index.html.haml]: You're converting a number to a string and then back into an integer on this line.

You should be able to use `garden.width.times do` (and the same with height, below).

**Comment by elizamarcum** [comment]
@ccampb13 and @amberagard There's just one small change to make for this one.

**Comment by elizamarcum** [comment]
@asejdina ^

**Comment by ccampb13** [comment]
@elizabrock @asejdina @amberagard corrected as requested!

**Pull request merged**